src/CMakeLists.txt

```diff
@@ -69,6 +69,7 @@ target_link_libraries(
   Shlwapi
   taskschd
   Version
+  runtimeobject
```

Comment:
This is needed by WIL's error macros, and was previously pulled in transitively from the d3dcompiler pragma lib